cmd/ddev/cmd/composer-create.go

```diff
@@ -119,22 +117,26 @@ project root will be deleted when creating a project.`,
 			composerCmd = append(composerCmd, "--no-interaction")
 		}
 
+		if preferDistArg {
+			composerCmd = append(composerCmd, "--prefer-dist")
+		}
+
 		composerCmdString := strings.TrimSpace(strings.Join(composerCmd, " "))
 		output.UserOut.Printf("Executing composer command: %s\n", composerCmdString)
 		stdout, _, err := app.Exec(&ddevapp.ExecOpts{
```

Comment:
This really should grab the stderr and report it if there is any. Alternately, you can explicitly set ExecOpts.NoCapture and let it just go out to them. If you think Exec() isn't handling stderr correctly, that's another thing and we can open an issue for it.